i18n/ko.toml

```diff
@@ -0,0 +1,204 @@
+
+[get_upgrade_title]
+other = "[Upgrading from v2 to v3?](https://helm.sh/docs/topics/v2_v3_migration/)"
+
+[get_upgrade_desc]
+other = "[Read the migration doc for more details.](https://helm.sh/docs/topics/v2_v3_migration/)"
+
+## homepage community
+[community_title]
+other = "Join the Community"
+
+[community_desc]
+other = "More information about the Helm project, and how to contribute."
+
+[community_events_title]
+other = "Events"
+
+[community_upcoming_events_title]
+other = "Upcoming Events"
+
+[community_upcoming_events_desc]
+other = "Watch this space..."
+
+[community_past_events_title]
+other = "Upcoming Events"
+
+[community_past_event_01]
+other = "_Feb 25th 2020_ - [Helm Security Webinar](https://www.cncf.io/webinars/helm-security-a-look-below-deck/)"
+
+[community_past_event_02]
+other = "_Nov 18th 2019_ - [KubeCon North America](https://events19.linuxfoundation.org/events/kubecon-cloudnativecon-north-america-2019/)"
+
+[community_past_event_03]
+other = "_Sep 11th 2019_ - [Helm Summit 2019](https://events19.linuxfoundation.org/events/helm-summit-2019/)"
+
+[community_sig_title]
+other = "SIG Apps"
+
+[community_sig_subtitle]
+other = "A Special Interest Group for deploying and operating apps in Kubernetes."
+
+[community_sig_desc]
+other = """
+[SIG-Apps](https://github.com/kubernetes/community/blob/master/sig-apps/README.md) is a Special Interest Group for deploying and operating apps in Kubernetes.
+
+They [meet each week](https://github.com/kubernetes/community/blob/master/sig-apps/README.md) to demo and discuss tools and projects. Community meetings are recorded and [shared to YouTube](https://www.youtube.com/results?search_query=kubernetes+sig+apps).
+"""
+
+[community_standup_title]
+other = "Developer Standups"
+
+[community_standup_time]
+other = "Thursdays 9:30-10am (PT)"
+
+[community_standup_desc]
+other = "These meetings are open to all. Check the [community repo](https://github.com/helm/community/blob/master/communication.md#meetings) for notes and details."
+
+[community_slack_title]
+other = "Slack"
+
+[community_slack_group_01_title]
+other = "Helm Users"
+
+[community_slack_group_01_desc]
+other = "Discussion around using Helm, working with charts and solving common errors."
+
+[community_slack_group_02_title]
+other = "Helm Development"
+
+[community_slack_group_02_desc]
+other = "Topics regarding Helm development, ongoing PRs, releases, etc."
+
+[community_slack_group_03_title]
+other = "Charts"
+
+[community_slack_group_03_desc]
+other = "Discussion for users and contributors to Helm Charts."
+
+[community_slack_join]
+other = "[Request access here](http://slack.k8s.io/) to join the Kubernetes Slack team."
+
+[community_contrib_title]
+other = "Contributing"
+
+[community_contrib_subtitle]
+other = "Helm always welcomes new contributions to the project!"
+
+
+[community_contrib_desc]
+other = """
+### Where to begin?
+
+Helm is a big project with a lot of users and contributors. It can be a lot to take in!s
```

Comment:
take in!s --> take in!

Reply:
Fixed. This was in the Chinese version and I fixed it there, too. Nice catch.